src/App.tsx

```diff
@@ -1,173 +1,79 @@
-                    <a
-                      className="has-text-danger"
-                      href="#/people/lievijne-jans-1542"
-                    >
-                      Lievijne Jans
-                    </a>
-                  </td>
-
-                  <td>f</td>
-                  <td>1542</td>
-                  <td>1582</td>
-                  <td>-</td>
-                  <td>-</td>
-                </tr>
-
-                <tr data-cy="person">
-                  <td>
-                    <a href="#/people/bernardus-de-causmaecker-1721">
-                      Bernardus de Causmaecker
-                    </a>
-                  </td>
-
-                  <td>m</td>
-                  <td>1721</td>
-                  <td>1789</td>
-
-                  <td>
-                    <a
-                      className="has-text-danger"
-                      href="#/people/livina-haverbeke-1692"
-                    >
-                      Livina Haverbeke
-                    </a>
-                  </td>
-
-                  <td>
-                    <a href="#/people/lieven-de-causmaecker-1696">
-                      Lieven de Causmaecker
-                    </a>
-                  </td>
-                </tr>
-              </tbody>
-            </table>
-          </div>
+export const App: React.FC = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isError, setIsError] = useState(false);
+  const [isLoading, setIsLoading] = useState(false);
+
+  useEffect(() => {
+    const loadPeople = async () => {
+      try {
+        setIsLoading(true);
+        const loadedPeople = await getPeople();
+
+        setPeople(loadedPeople);
+      } catch (error) {
+        setIsError(true);
+      } finally {
+        setIsLoading(false);
+      }
+    };
+
+    loadPeople();
+  }, []);
+
+  return (
+    <div data-cy="app">
+      <Navigation />
+      <main className="section">
+        <div className="container">
+          <Routes>
+            <Route
+              path="/"
+              element={<h1 className="title">Home Page</h1>}
+            />
+
+            <Route
+              path="home"
+              element={<Navigate to="/" replace />}
+            />
+            <Route path="people">
+              <Route
+                index
+                element={(
+                  <PeoplePage
+                    people={people}
+                    isLoading={isLoading}
+                    isError={isError}
+                  />
+                )}
+              />
+              <Route
+                path=":slug"
+                element={(
+                  <PeoplePage
+                    people={people}
+                    isLoading={isLoading}
+                    isError={isError}
+                  />
+                )}
+              />
+            </Route>
+
+            <Route
+              path="*"
+              element={<h1 className="title">Page not found</h1>}
+            />
```

Comment:
1. Let's create a component `Layout`. 
2. Wrap all your routes to the `layout route`. 
3. This layout component should render a `Navigation` and wrap the content into a section. You can use Outlet from react-router-dom.

```
  <div className="section">
    <Navigation />

    <Outlet />
  </div>
```

```suggestion
<Routes>
  <Route path="/" element={<Layout />}>
    .. homepage
    ...people
    .. etc
</Routes>     
```